package/evtest/package

```diff
@@ -10,17 +10,14 @@ timestamp=2020-12-30T02:52Z
 section="utils"
 maintainer="Linus K. <linus@cosmos-ink.net>"
 license=GPL-2.0-only
+makedepends=(build:automake)
 
 image=base:v1.2.1
-source=(https://gitlab.freedesktop.org/libevdev/evtest/-/archive/evtest-1.34/evtest-evtest-1.34.zip)
+source=("https://gitlab.freedesktop.org/libevdev/evtest/-/archive/evtest-${pkgver%-*}/evtest-evtest-${pkgver%-*}.zip")
```

Comment:
This change to 1.34-2 breaks it for me.

Reply:
In what way? The build seems to run fine.

Reply:
Interesting. It should resolve to the same 1.34. 
I have checked out the branch on another system and will double check.

Reply:
I have no idea why that system was not building properly, but on the new one it builds properly. Thanks for checking with me though.